Tasks/NuGetAuthenticateV1/_buildConfigs/Node20/package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "azure-pipelines-tasks-artifacts-common": "^2.212.0",
+    "azure-pipelines-tasks-artifacts-common": "^2.243.0",
     "azure-pipelines-task-lib": "^4.0.0-preview",
```

Comment:
Ive been told to upgrade the task-lib with the common library.

Reply:
Here and for the wif package json

Reply:
upgraded version in both spots!